engine-tests/src/tests/erc20_connector.rs

```diff
@@ -482,6 +494,104 @@ pub mod sim_tests {
         );
     }
 
+    #[test]
+    fn test_exit_to_near_wnear_unwrapped() {
+        // Deploy Aurora; deploy wnear; bridge wnear to ERC-20 on Aurora
+        let TestExitToNearContext {
+            ft_owner,
+            ft_owner_address,
+            aurora,
+            wnear,
+            wnear_erc20,
+            ..
+        } = test_exit_to_near_common();
+
+        let ft_owner_balance = get_account_balance(ft_owner.account_id.as_str(), &aurora);
+
+        // Call exit function on ERC-20; observe ERC-20 burned + near tokens transferred
+        let result = exit_to_near(
+            &ft_owner,
+            &format!("{}:unwrap", ft_owner.account_id.as_str()),
+            FT_EXIT_AMOUNT,
+            &wnear_erc20,
+            &aurora,
+        );
+        let total_tokens_burnt: u128 = result
+            .promise_results()
+            .iter()
+            .map(|r| r.as_ref().unwrap().outcome().tokens_burnt)
+            .sum();
+
+        // Check that the wnear tokens are properly unwrapped and transferred to `ft_owner`
+        assert_eq!(
+            get_account_balance(ft_owner.account_id.as_str(), &aurora),
+            ft_owner_balance - total_tokens_burnt + FT_EXIT_AMOUNT
```

Comment:
I'm new to this part of the code base and I'm missing some context. What does it mean for tokens to be burned here?

Reply:
It is the amount of native near tokens burnt corresponding to the burnt gas amount.